src/skmultiflow/trees/nodes/lc_inactive_learning_node.py

```diff
@@ -0,0 +1,17 @@
+from skmultiflow.trees.nodes import InactiveLearningNode
+
+
+class LCInactiveLearningNode(InactiveLearningNode):
```

Comment:
Please add a short description.

Reply:
Description added in commit [5f331dc](https://github.com/scikit-multiflow/scikit-multiflow/pull/149/commits/5f331dce48293665f7ef78b059ba2d6782606438).